config/owasp/suppressions.xml

```diff
@@ -12,12 +12,4 @@
 <cve>CVE-2021-22112</cve>
 <cve>CVE-2022-22976</cve>
 </suppress>
```

Comment:
Having the Temporary Suppression commented out causes the Suppressions workflow to fail.

The workflow generates the section when required.